tensorflow/lite/micro/tools/tflite_flatbuffer_align_wrapper.cc

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2023 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <pybind11/pybind11.h>
+#include <pybind11/pytypes.h>
+
+#include "flatbuffers/util.h"
+#include "tensorflow/lite/schema/schema_generated.h"
+
+namespace py = pybind11;
+
+void align_tflite_model(const char* input_file_name,
+                        const char* output_file_name) {
+  std::string model_file;
+  // Read the file into a string using the included util API call:
+  flatbuffers::LoadFile(input_file_name, false, &model_file);
+  fprintf(stderr, "Original model size: %lu\n", model_file.size());
```

Comment:
let's move the printf out into the py_binary. I'm also ok with removing the printf completely.

Reply:
Done.